overlay_gl/CMakeLists.txt

```
@@ -96,6 +96,11 @@ if(NOT APPLE)
 					"-ldl"
 					"-lrt"
 			)
+		endif()
+	endif()
+
+	if(${CMAKE_SYSTEM_NAME} STREQUAL "Linux" OR ${CMAKE_SYSTEM_NAME} STREQUAL "FreeBSD")
```

```suggestion
	if(UNIX AND NOT APPLE)
```

macOS is the only (actively developed) operating system without 32 bit support, as far as I'm aware.

---

The OS check seems unnecessary then, since this is already in the NOT APPLE block. Just checking that the overlay_gl_x86 target exists should be enough, but I wasn't sure if there was some reason that the 32-bit library was only installed on Linux and was playing more cautious by opting-in FreeBSD.

---

Oh, I see. Checking whether the target exists is ideal.